src/Microsoft.Azure.WebJobs.Host/Bindings/Runtime/Binder.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.WebJobs
     /// <summary>
     /// Class providing functionality for dynamically binding to WebJobs SDK attributes at runtime.
     /// </summary>
-    public class Binder : IBinder, IWatchable, IDisposable
+    public class Binder : IWatchable, IDisposable
```

Comment:
The problem is that interfaces are not extensible at all. That's exactly the problem I ran into with IBinder - for Functions I needed to add a new overload as well as access to BindingData but was unable to. Having a concrete class allows us to add new members in the future as needed without breaking changes.